spec/moonshine_generator_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+eequire 'spec_helper'
```

Comment:
@uhlenbrock No, thanks it is fixed now. This should be rolling out later today, just need to fix Facter 2 compatibility in the popular moonshine plugins.